### ISSUE TITLE
Progress View

### ISSUE THREAD

**Issue by lonkelle** [SideStore/SideServer-macOS#5]
Hey @rrroyal - you can send your Progress View PR straight here. No rush since it's just a QOL thing.

**Comment by f1shy-dev** [comment]
Wrong mention i'm assuming? 😅

**Comment by lonkelle** [comment]
Yeah, I fixed it immediately after and tagged @rrroyal but it was too late at that point. 🤦‍♀️

**Comment by f1shy-dev** [comment]
Eh, we all make mistakes

GitHub Mobile is just too fast with the push notifications 😅
